src/pykiso/auxiliary.py

```diff
@@ -46,6 +47,11 @@ def __init__(self) -> None:
         self.is_instance = False
         self.stop_event = None
         self._aux_copy = None
+        # add internal kiso log levels in case the auxiliary is used without pykiso cli
+        if not hasattr(logging, "INTERNAL_WARNING"):
```

Comment:
Valid comment of codecov, this is not tested yet

Reply:
Now it is